cmd/proxy/main.go

```diff
@@ -8,6 +8,14 @@ import (
 )
 
 func main() {
+	go func() {
+		handler := &proxy.HealthzProxy{}
+		log.Println("Starting health check on :8080")
+		if err := http.ListenAndServe(":8080", handler); err != nil {
```

Comment:
health check generally listens on the loopback interface, i.e. localhost

Reply:
http health check is listens on the pod ip, so no change here